go/coinstacks/cosmos/api/handlers.go

```diff
@@ -193,3 +193,7 @@ func (h *Handler) GetTxHistory(pubkey string, cursor string, pageSize int) (api.
 func (h *Handler) SendTx(rawTx string) (string, error) {
 	return h.httpClient.BroadcastTx([]byte(rawTx))
 }
+
+func (h *Handler) EstimateGas(rawTx string) (string, error) {
+	return h.httpClient.GetEstimateGas([]byte(rawTx))
```

Comment:
```suggestion
	return h.httpClient.GetEstimateGas(rawTx)
```

Reply:
I have change this. 
But remember that `Use a query parameter for txBytes` is in the https://github.com/shapeshift/unchained/issues/234